### ISSUE TITLE
Ruby attr_accessor

### ISSUE THREAD

**Issue by AlexWayfer** [faceair/atom-goto-definition#29]
Hello!

It would be nice to support the `attr_accessor`, `attr_reader` and `attr_writer` in Ruby.

This works as expected:

``` ruby
class Foo
  def bar
  end

  def test
    bar
  end
end
```

But this doesn't:

``` ruby
class Foo
  attr_accessor :bar # or attr_reader

  def test
    bar
  end
end
```

**Comment by tracyloisel** [comment]
Hi @AlexWayfer, i will make a PR tomorrow to add attr_x and scope definition

**Comment by tracyloisel** [comment]
https://github.com/faceair/atom-goto-definition/pull/37

**Comment by tracyloisel** [comment]
https://medium.com/map-tech-corner/improve-atom-goto-definition-package-4fc1298a38d3#.gl2bfhdrs

**Comment by faceair** [comment]
Solved by #37
